apps/spkac.c

```diff
@@ -138,8 +138,16 @@ int spkac_main(int argc, char **argv)
         if (challenge != NULL)
             ASN1_STRING_set(spki->spkac->challenge,
                             challenge, (int)strlen(challenge));
-        NETSCAPE_SPKI_set_pubkey(spki, pkey);
-        NETSCAPE_SPKI_sign(spki, pkey, EVP_md5());
+        i = NETSCAPE_SPKI_set_pubkey(spki, pkey);
+        if (i <= 0) {
```

Comment:
`NETSCAPE_SPKI_set_pubkey` only ever returns 0 or 1, so this could just be written as:

````
if (!NETSCAPE_SPKI_set_pubkey(spki, pkey))
````